scripts/zones/Jugner_Forest/npcs/qm2.lua

```diff
@@ -12,9 +14,10 @@ end
 entity.onTrigger = function(player, npc)
     local phase = VanadielMoonPhase()
     local direction = VanadielMoonDirection()
-    if player:getCharVar("sinHunting") == 4 and
-    ((phase <= 90 and direction == 2) or (phase <= 95 and direction == 1)) then
+    if player:getCharVar("sinHunting") == 4 and IsMoonFull() then
```

Comment:
Can clean up phase and direction now then, yea?

Reply:
IsMoonFull() handles phase and direction:
```cpp
case 0: // None
if (phase == 100)

case 1: // Waning (decending)
if (phase >= 95 && phase <= 100)

case 2: // Waxing (increasing)
if (phase >= 90 && phase <= 100)
```

Reply:
Yea, lines 15 and 16 should be able to be removed with your PR

Reply:
It's gone.

Reply:
It looks like you might need to push your work up to your branch if you've deleted it. I'm still seeing those lines here and its causing the checks to fail.